SkillSwapHub/src/main/resources/static/js/registerUser.js

```
@@ -43,15 +43,15 @@
     messageBox.innerHTML = ""; // Clear previous messages
 
     for (let key in errors) {
-        let errorMessage = `<p style="color: red;">${errors[key]}</p>`;
+        let errorMessage = `<p class="validation-error">${errors[key]}</p>`;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'errorMessage' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)